vespa-http-client/src/test/java/com/yahoo/vespa/http/client/core/operationProcessor/OperationProcessorTest.java

```diff
@@ -410,4 +412,18 @@ public void testSendsResponseToQueuedDocumentOnClose() throws InterruptedExcepti
         operationProcessor.close();
         countDownLatch.await();
     }
+
+    @Test
+    public void testUnknownHost() {
+        try {
+            new SessionParams.Builder()
+                    .addCluster(new Cluster.Builder().addEndpoint(Endpoint.create("localhost")).build())
+                    .addCluster(new Cluster.Builder().addEndpoint(Endpoint.create("unknown")).build())
+                    .build();
+        }
+        catch (IllegalArgumentException e) {
+            assertEquals("", e.getMessage());
```

Comment:
Based on the code, shouldn't this be `"Unknown host: unknown"`?

Reply:
Yes, fixed.